firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -185,6 +186,18 @@ int stillness_timer_set(String input) {
     return returnFlag;
 }
 
+// returns the previous length of the stillness timer
+int reset_stillness_timer_for_alerting_session(String input) {
```

Comment:
I wasn't aware that we are adding a cloud function for resetting the stillness timer. Is this just the easiest way for the server to tell the firmware to zero the `state3_stillness_timer` variable?

Reply:
As far as I know, using a cloud function via the API is the only way that we can communicate from the server to the firmware.